### DISCUSSION TITLE
Error writing a text object that contains an umlaut - 'ā'

### DISCUSSION THREAD

**Discussion by thabrlawrenz** [gettalong/hexapdf#331] [General]
Hi, you may have answered this before, but I just cant get this working.
I am using composer
```
composer = HexaPDF::Composer.new(page_size: :A4, margin: page_margin)
```
 then when I try and draw a text object with text containing "ā" using ```HexaPDF::Layout::TextFragment.create(text_title ...```

I receive this error:
```
HexaPDF::Error (Used Type1 encoding has no codepoint for :amacron)
```

I have tried all sorts of config options to make this work, just cant get it to draw text that contains characters with umlauts: ā, ö.
An suggestions please?

**Comment by gettalong** [comment]
First, I'm not sure why you are manually creating a TextFragment instance. If you are using the composer, you can just write `composer.text(...)`. Alternatively, it is better to use `composer.document.layout.text_fragments` so that you get all benefits, like fallback font support.

As for the error itself: You need to use a font that has glyphs for the codepoints you want to show. It seems that you are using one of the built-in standard PDF fonts. They only provide a very small subset of glyphs. It is better to use a TrueType font. See e.g. the [composer tutorial](https://hexapdf.gettalong.org/documentation/document-creation/composer-tutorial.html) for details.

**Reply by thabrlawrenz** [reply]
Yep ok that makes sense. I tried with loading the font map as indicated in the composer tutorial with NotoSans and it worked great. Thanks for your help.
